### DISCUSSION TITLE
Is there an end to optimization?

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/roslyn#66439] [General]
With our invention : The AI, getting advanced each day with its capability to self evolve, what's our endgame
Can we find a way to do things better, a way to optimize codes so it's impossible to be optimized any further, before AI does a better job than we do approaching this limit (is there a limit) through Newton's method
If our job security is count on finishing things that are left unfinished, what's to say our brain won't actually get comfortable then become stupid then maybe it is possible AI takes our job in the end, shouldn't we put our heads together like many of us that walked in front of us did to figure this out

**Comment by CyrusNajmabadi** [comment]
@Xyncgas these discussions are off-topic fur this repo. I'm the future please keep things specific and appropriate for the C# Language/Compiler domain. 

There have been several posts recently from you like this. They've caused some head scratching, but have mainly been ignored/locked. However, the rate you've been posting stuff like this has increased and we'd really liked you to stop so as to keep you a community member in good standing here. 

Thanks!
